EnvironmentExtensions still referenced in v5.0.0

Looks like I missed something in #103 as I get an error when trying to use v5.0.0:
```
Compiling build script...
error CS0234: The type or namespace name 'EnvironmentExtensions' does not exist in the namespace 'Cake.Incubator' (are you missing an assembly reference?)
Error: Cake.Core.CakeException: Error(s) occurred when compiling build script:
error CS0234: The type or namespace name 'EnvironmentExtensions' does not exist in the namespace 'Cake.Incubator' (are you missing an assembly reference?)
   at Cake.Scripting.Roslyn.RoslynScriptSession.Execute(Script script)
   at Cake.Core.Scripting.ScriptRunner.Run(IScriptHost host, FilePath scriptPath, IDictionary`2 arguments)
   at Cake.Commands.BuildCommand.Execute(CakeOptions options)
   at Cake.CakeApplication.Run(CakeOptions options)
   at Cake.Program.Main()
```

--- a/src/Cake.Incubator/Cake.Incubator.csproj
+++ b/src/Cake.Incubator/Cake.Incubator.csproj
@@ -45,9 +45,6 @@
             <_Parameter1>Cake.Incubator.EnumerableExtensions</_Parameter1>
         </AssemblyAttribute>
         <AssemblyAttribute Include="Cake.Core.Annotations.CakeNamespaceImportAttribute">
-            <_Parameter1>Cake.Incubator.EnvironmentExtensions</_Parameter1>
-        </AssemblyAttribute>
-        <AssemblyAttribute Include="Cake.Core.Annotations.CakeNamespaceImportAttribute">
             <_Parameter1>Cake.Incubator.FileExtensions</_Parameter1>
         </AssemblyAttribute>
         <AssemblyAttribute Include="Cake.Core.Annotations.CakeNamespaceImportAttribute">

